story.py

```diff
@@ -31,7 +31,19 @@ def read_tsv():
 	tsv_file.close()
 
 def read_user_input():
-	user_input = input()
+	choices = '0'
+	prompt = None
+	ending = "Press any key to exit."
+	while prompt != 'The end.\n' + ending:
+		story = prev_choices_dict[choices]
+		prompt = stories[story]+"\n"
+		print(story)
+		if prompt == 'The end.\n':
+			prompt += ending
+		if prompt == 'n/a\n':
+			choices = next_choices_dict[story]
+		else:
+			choices += "." + input(prompt)
```

Comment:
right now, if the user enters invalid input, the program crashes. you want to be able to repeatedly ask for a valid response until one is provided or the user quits.